src/PaymentGateways/PayPalStandard/Actions/ProcessIpnDonationRefund.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Give\PaymentGateways\PayPalStandard\Actions;
+
+use Give\ValueObjects\Money;
+use Give_Payment;
+
+/**
+ * @unreleased
+ */
+class ProcessIpnDonationRefund
+{
+    /**
+     * @unreleased
+     *
+     * @param array $ipnEventData
+     * @param Give_Payment $donation
+     *
+     * @return void
+     */
+    public function __invoke(array $ipnEventData, Give_Payment $donation)
```

Comment:
As we discussed in Stripe, I thought that we had decided not to leverage `Give_Payment` class in the new gateways? Instead, in Stripe, we are using the existing Give Payment API methods, like `give_insert_payment_note( $payment_id, $note )`.

Reply:
@kjohnson I updated the code to use it internally. instead of passing as a function argument.